packages/core-bridge/index.d.ts

```diff
@@ -116,20 +119,33 @@ export interface WorkerOptions {
   maxCachedWorkflows: number;
 }
 
+export interface CoreLog {
+  /** Log message */
+  message: string;
+  /** Unix millis since epoch. Ideally would be bigint but Neon can't send those
+   */
+  // TODO: Not meaningfully usable until logger interface has a timestamp concept
+  timestamp_millis: number;
+  /** Log level */
+  level: string;
```

Comment:
Use this: https://github.com/temporalio/sdk-node/blob/dfb1c3dfc7eace3e8ec5d3776b4f671f6633bc57/packages/worker/src/logger.ts#L11